src/Sulu/Bundle/ReferenceBundle/Domain/Model/ReferenceInterface.php

```diff
@@ -0,0 +1,75 @@
+<?php
+
+/*
+ * This file is part of Sulu.
+ *
+ * (c) Sulu GmbH
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Sulu\Bundle\ReferenceBundle\Domain\Model;
+
+interface ReferenceInterface
+{
+    public const RESOURCE_KEY = 'references';
+    public const LIST_KEY = 'references';
+
+    public function getId(): int;
+
+    public function getResourceKey(): string;
+
+    public function setResourceKey(string $resourceKey): static;
+
+    public function getResourceId(): string;
+
+    public function setResourceId(string $resourceId): static;
+
+    public function getReferenceLocale(): string;
```

Comment:
should we make this nullable if we want e.g. show contact or accounts in future in the reference list of media? /cc @Prokyonn

Reply:
yes makes sense, it is then also easier for custom non translatable entities to integrate references